lib/languages/nim.txt

```diff
@@ -0,0 +1,5 @@
+import
+from 
+export
```

Comment:
```suggestion
export
include
```

We have `import`, so maybe we want to ignore lines starting with `include` too? I don't know.

Reply:
Yup, forgot this